datafusion/core/tests/user_defined/user_defined_plan.rs

```diff
@@ -102,362 +96,10 @@ use datafusion_physical_plan::execution_plan::{Boundedness, EmissionType};
 use async_trait::async_trait;
 use futures::{Stream, StreamExt};
 
-/// Execute the specified sql and return the resulting record batches
```

Comment:
Given that we'll move the code about "how to write the user defined plan" to doc, so I moved the useless tests in the PR

Reply:
Good idea